### DISCUSSION TITLE
Taking filebrowser to the moon 🚀

### DISCUSSION THREAD

**Discussion by thewh1teagle** [filebrowser/filebrowser#2183] [Ideas]
@o1egl 
I think that we should add the ability to make extensions for `filebrowser`
this way, we can continue to keep it simple as possible, but for someone who want to add extension it will be maybe available somewhere in the settings (something like extensions marketplace)
I think it's pretty important feature. 
some extensions that can takes this thing to the moon 🚀
- API service for 3rd integrations
- Remote folders via sftp / ftp etc
- Android App that sync between (Google Drive alternative)
- Dlna, smb
- Remote access via upnp / 3rd party services

**Comment by ramiresviana** [comment]
The interface also needs some attention, my concept made with Material Design 3:

### Filebrowser Theme
![filebrowser-material-3](https://user-images.githubusercontent.com/59319979/198905877-7d46f4e9-fabe-4dae-8d04-eefb6d2dd4be.png)

![filebrowser-material-3-dark](https://user-images.githubusercontent.com/59319979/198905880-901c8392-7ac3-4467-93fc-ef45c230df3d.png)

### User Colors
![filebrowser-material-3-custom](https://user-images.githubusercontent.com/59319979/198905888-46477543-8d45-4773-a743-a0e6357fbcc0.png)

![filebrowser-material-3-custom-dark](https://user-images.githubusercontent.com/59319979/198905892-13d52517-daa9-4261-be8d-e8426c735774.png)

**Reply by Silloky** [reply]
Nice ! I don't really like the green dark mode though.

**Reply by AC-Lover** [reply]
How do I use these themes?

**Reply by ramiresviana** [reply]
This is just a concept, it was not implemented.
